addons/im_livechat/static/src/core/web/discuss_patch.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="im_livechat.Discuss" t-inherit="mail.Discuss" t-inherit-mode="extension">
+        <xpath expr="//*[@t-ref='root']" position="before">
+            <ControlPanel t-if="store.discuss.fromLivechatMenu"/>
```

Comment:
I'd argue we should always have the control panel in discuss, it's annoying to lose the breadcrumbs.